diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9c5b037..fe5f8d0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -57,10 +57,6 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/lib")
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/bin")
 set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/lib")
 
-if (WIN32)
-	set(CMAKE_INSTALL_PREFIX ${PROJECT_BINARY_DIR}/dist)
-endif()
-
 # setup global include_directories (for backwards compatibility)
 include_directories("${CMAKE_BINARY_DIR}/gen/")
 include_directories("${CMAKE_BINARY_DIR}/gen/jsonrpccpp/common")
diff --git a/src/jsonrpccpp/CMakeLists.txt b/src/jsonrpccpp/CMakeLists.txt
index d22a2e1..af172a1 100644
--- a/src/jsonrpccpp/CMakeLists.txt
+++ b/src/jsonrpccpp/CMakeLists.txt
@@ -153,13 +153,6 @@ install(FILES ${jsonrpc_install_header_server}  DESTINATION include/jsonrpccpp/s
 install(FILES ${client_connector_header}        DESTINATION include/jsonrpccpp/client/connectors)
 install(FILES ${server_connector_header}        DESTINATION include/jsonrpccpp/server/connectors)
 
-if (WIN32)
-	file(GLOB win32_libs ${CMAKE_SOURCE_DIR}/win32-deps/lib/*.dll)
-	file(COPY ${win32_libs} DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
-	install(FILES ${win32_libs} DESTINATION bin)
-	install(DIRECTORY ${CMAKE_SOURCE_DIR}/win32-deps/include DESTINATION .)
-endif()
-
 install(TARGETS ${ALL_LIBS}
         LIBRARY DESTINATION lib/${CMAKE_LIBRARY_PATH}
         ARCHIVE DESTINATION lib/${CMAKE_LIBRARY_PATH}
